netlify.toml

```
@@ -11,7 +11,7 @@
   publish = "dist/"
 
   # Default build command.
-  command = "npm run build"
+  command = "echo '@fortawesome:registry=https://npm.fontawesome.com/' > .npmrc && echo '//npm.fontawesome.com/:_authToken=${FA_TOKEN}' >> .npmrc && npm install && npm run build"
```

I guess I imagined this for readability -- whatever works though :D

```suggestion
  command = """
      echo '@fortawesome:registry=https://npm.fontawesome.com/' > .npmrc && 
      echo '//npm.fontawesome.com/:_authToken=${FA_TOKEN}' >> .npmrc && 
      npm install && npm run build"
  """
```

---

Weird github syntax highlighting makes this scary